scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/ClientHandshaker.java

```diff
@@ -471,6 +471,7 @@ private void receivedServerHelloDone(ServerHelloDone message) throws HandshakeEx
 			generateKeys(premasterSecret);
 			break;
 		case PSK:
+			recordLayer.pauseRetransmission();
```

Comment:
> And we may move this more to the upper, say also for "expensive asymetric" cipher suites. 

My idea, was to also pause for other potential expensive operations.
But the right places would require more research :-).